### PR TITLE
Add Badge to Pill's related components

### PR DESCRIPTION
**Pull request by mlmsmith**
This change is related to #25, adding a link in the right-hand column of the pill component back to badge

### CODE REVIEW

**Comment by joshwooding** [comment]
Hi @mlmsmith, thanks for the contribution. We require all contributors to JPMC projects to complete our Contribution Licence Agreement (CLA). You can sign as an individual or as representing your company. You only need to do this once and it covers all current and future projects managed under our GitHub organization. 

The contribution agreement can be found here [JPMC CLA](https://github.com/jpmorganchase/.github/blob/main/jpmc-cla-20230406.md), follow the instructions to complete and return via email.

**Comment by mlmsmith** [comment]
Hi Josh,

Thank you for your email. I have completed and submitted the CLA, just waiting for it to be processed.

Kind regards

Michael

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by joshwooding** [comment]
> Hi Josh, Thank you for your email. I have completed and submitted the CLA, just waiting for it to be processed. Kind regards Michael Sent from Outlook for iOS<[aka.ms/o0ukef](https://aka.ms/o0ukef)>
> […](#)

Thanks for the quick reply

**Comment by joshwooding** [comment]
I'll move this to draft as we're waiting for the CLA approval.

**Comment by brooklynrob** [comment]
@joshwooding -- @mlmsmith's CLA is approved (CC @javasun1000).

**Comment by joshwooding** [comment]
> @joshwooding -- @mlmsmith's CLA is approved (CC @javasun1000).

Thanks for the update @brooklynrob !

**Review by joshwooding — APPROVED** [review]

**Review by bhoppers2008 — APPROVED** [review]

**Pull request merged**
